source/theme.css

```diff
@@ -303,7 +303,7 @@ header .header__searchbar {
 		display: none;
 	}
 =======
-	
+
 >>>>>>> c8fd417... Added a search box to the nav header for the mobile view.
```

Comment:
This looks like a bad merge?

Reply:
@coreyhulen Good catch, will submit a separate PR for it, looks like it's from an old PR